pkg/reconciler/kubernetes/tektonpipeline/tektonpipeline.go

```diff
@@ -105,7 +106,7 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, tp *v1alpha1.TektonPipel
 		return err
 	}
 	stages := common.Stages{
-		common.AppendTarget,
+		r.appendPipelineTarget,
```

Comment:
shall we keep these 2 as separate stages. ie, leave common.AppendTarget as it is. Then make r.appendPipelineTarget next stage and make it handle only what `addProxy` is doing.

In addition, after separating we can also rename the `r.appendPipelineTarget` to something like `r.appendProxy`

Reply:
I will work on this in a follow-up PR. Thanks